### PR TITLE
Add operation to get travellers trend

### PR DESCRIPTION
**Pull request by gilbertotcc**
Add the description of the operation that allows to get the trends of
the passengers in an airport.

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   },
   "MD013": {
     "line_length": 80,
-    "code_blocks": false
+    "code_blocks": false,
+    "tables": false
```
Comment (iamleot) [inline .markdownlint.json:9]: Please put such change in a separate commit next times (it is unrelated to the change in README.md and it would be nice to have a rationale why check on tables would disabled if we would like to look at `.markdownlint.json` history on its own).

**Review by iamleot — APPROVED** [review]

File: .markdownlint.json (hunk 1 above)
Comment (gilbertotcc) [reply .markdownlint.json:9]: You're right. I will keep them separated in next PRs.

**Review by Giackgamba — APPROVED** [review]

**Review by PepeExpress — APPROVED** [review]

**Review by yonnisc — APPROVED** [review]
